src/WebJobs.Script.Analyzers/WebJobsAttributeAnalyzer.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Functions.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class WebJobsAttributeAnalyzer : DiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(DiagnosticDescriptors.IllegalFunctionName); } }
+
+        public static void VerifyWebJobsLoaded()
+        {
+            // Check if WebJobs types can be loaded
+            var jobHost = new JobHost(new OptionsWrapper<JobHostOptions>(new JobHostOptions()), null);
+        }
+
+        public override void Initialize(AnalysisContext context)
+        {
+            // https://stackoverflow.com/questions/62638455/analyzer-with-code-fix-project-template-is-broken
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
+            context.EnableConcurrentExecution();
+
+            VerifyWebJobsLoaded();
+
+            // Analyze method signatures.
+            context.RegisterSyntaxNodeAction(AnalyzeMethodDeclarationNode, SyntaxKind.MethodDeclaration);
+
+            // Hook compilation to get the assemblies' references and build the WebJob tooling interfaces.
+            context.RegisterCompilationStartAction(AnalyzeCompilation);
```

Comment:
Do not think we need this one either in this analyzer PR as this is not really needed for function name validation analyzer.

Reply:
True; this analyzer doesn't use the tooling object. Thanks for the catch.